### PR TITLE
Adding html to string conversion

### PR DESCRIPTION
**Pull request by PichapopRo**
## What I have added.
- Adding HTML conversion to fix the fetched description came in HTML format.

### CODE REVIEW

File: webpage/modules/proxy.py
Comment (knilios) [inline webpage/modules/proxy.py:1]: The queryset type hint is gone.

File: webpage/modules/builder.py
Comment (knilios) [inline webpage/modules/builder.py:1]: The__strip_html method has no type hint and is an incomplete docstring.

**Review by knilios — CHANGES_REQUESTED** [review]
Mostly an issue with docstring and type hint.

**Review by SariyaP — APPROVED** [review]
Looks ok

**Review by knilios — APPROVED** [review]
OK

**Pull request merged**